cosmo_tester/test_suites/test_blueprints/nodecellar_test.py

```diff
@@ -81,6 +81,27 @@ def _get_wines_request(self):
         return requests.get('http://{0}:{1}/wines'.format(
             self.public_ip, self.nodecellar_port))
 
+    def get_public_ip(self, nodes_state):
+        public_ip = None
+        entrypoint_node_name = self.entrypoint_node_name
+        entrypoint_runtime_property_name = self.entrypoint_property_name
+        for key, value in nodes_state.items():
+            if '_host' in key:
```

Comment:
didn't notice this part before. it is used to make assertions on the expected host node. please refactor in another way so that get_public_ip wont implicitly make assertions.